driver/test_kubeflow_workloads.py

```diff
@@ -34,6 +39,15 @@
 
 PYTEST_CMD_BASE = "pytest"
 
+PODDEFAULT_RESOURCE = create_namespaced_resource(
+    group="kubeflow.org",
+    version="v1alpha1",
+    kind="poddefault",
+    plural="poddefaults",
+)
+PODDEFAULT_WITH_PROXY_PATH = Path("tests") / "proxy-poddefault.yaml.j2"
+PODDEFAULT_WITH_PROXY_NAME = "notebook-proxy"
```

Comment:
we can parse the name from the yaml rather

Reply:
Removed the var, I have used a diff method. Let me know what you think.